### ISSUE TITLE
Simplified routing syntax

### ISSUE THREAD

**Issue by designermonkey** [symphonists/url_router#11]
I would like to table a debate about the syntax of the simplified url routing that @alpacaaa kindly added. I am leaning towards this now for general use, and it should be definitely pushed as the primary routing technique over the more 'advanced' regex.

I feel personally that as primarily XSL developers, who are used to parameter syntax like `$param`, it can be confusing to use syntax from Rails like `:param`.

Any thoughts? @nils-werner @alpacaaa ?

**Comment by bauhouse** [comment]
That's a good point @designermonkey. I agree with you. For the sake of consistency, we should use the parameter syntax: `$param`. It is very likely that this could be implemented as a non-breaking update by allowing both.

**Comment by tonyarnold** [comment]
tl;dr; BOOM! Needs testing, but have at it.

**Comment by alpacaaa** [comment]
That's something I haven't thought about, thanks for implementing it Tony :)
